README.md

```diff
@@ -103,8 +103,8 @@ All parameters default to true, which means:
 1. `setEnvVars`, first arg: `true` in order to actually update env vars.
  Setting it `false` allows consumers of this library to process the .env file
  but use it for things other than updating env vars, as a generic configuration file.
- The Load methods all return an `IEnumerable<KeyValuePair<string,string>> for this, but
- there is an extension method ToDictionary to get a dict with the last value for each key.
+ The Load methods all return an `IEnumerable<KeyValuePair<string,string>>` for this, but
+ there is an extension method `ToDictionary(CreateDictionaryOption.TakeLast)` to get a dict with the last value for each key.
```

Comment:
You can and should elaborate on the new options now.

Reply:
Added a short info, guess more is not needed or at least confusing.